src/jit-analyze/JitAnalyzeRootCommand.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+
+namespace ManagedCodeGen
+{
+    internal sealed class JitAnalyzeRootCommand : RootCommand
+    {
+        public Option<string> BasePath { get; } =
+            new(new[] { "--base", "-b" }, "Base file or directory");
+        public Option<string> DiffPath { get; } =
+            new(new[] { "--diff", "-d" }, "Diff file or directory");
+        public Option<bool> Recursive { get; } =
+            new(new[] { "--recursive", "-r" }, "Search directories recursively");
+        public Option<string> FileExtension { get; } =
+            new("--file-extension", _ => ".dasm", true, "File extension to look for");
+        public Option<int> Count { get; } =
+            new(new[] { "--count", "-c" }, _ => 20, true, "Count of files and methods (at most) to output in the summary. (count) improvements and (count) regressions of each will be included");
+        public Option<bool> Warn { get; } =
+            new(new[] { "--warn", "-w" }, "Generate warning output for files/methods that only exists in one dataset or the other (only in base or only in diff)");
+        public Option<List<string>> Metrics { get; } =
+            new(new[] { "--metrics", "-m" }, _ =>  new List<string> { "CodeSize" }, true, $"Comma-separated metric to use for diff computations. Available metrics: {MetricCollection.ListMetrics()}");
```

Comment:
Is it still true that this can be specified with comma-separated syntax?

Reply:
Note that superpmi.py in dotnet/runtime is passing this arg so it will need changes too.

It does introduce some weirdness if you try to run superpmi on the servicing branches, but I'm not sure if we really need to worry about it (running superpmi.py on servicing branches is rare, and so is specifying custom metrics to superpmi.py). I would be fine with changing the syntax here, but also cc @dotnet/jit-contrib for a chance to express their concerns.

Reply:
Do you want to submit the fix for this too? Specifically around here: https://github.com/dotnet/runtime/blob/db717e30839c532cad5b269ac11c5d2c91dad639/src/coreclr/scripts/superpmi.py#L1790-L1791
Looks like the help message for the `-metrics` arg in superpmi.py needs an update too.